packages/edit-site/src/components/welcome-guide/styles.js

```diff
@@ -117,7 +117,7 @@ export default function WelcomeGuideStyles() {
 							</h1>
 							<p className="edit-site-welcome-guide__text">
 								{ __(
-									'New to block themes and styling your site?'
+									'New to block themes and styling your site? '
```

Comment:
I think this needs a translators comment?, because that space can be very easy to miss.